src/TurbulenceStatistics/viscous_dissipation.jl

```diff
@@ -21,6 +21,21 @@ kinetic energy of `model`.
 
 Calling `compute!(ϵ::ViscousDissipation)` computes the viscous dissipation of
 turbulent kinetic energy associated with `model` and stores it in `ϵ.data`.
+
+The viscous dissipation ``\epsilon`` is defined
+
+```math
+ϵ ≡ ∫ uⱼ ∂ⱼ τᵢⱼ \, \rm{d} V
+```
```

Comment:
Tests are failing and complaining about invalid escape sequences. I think we need to escape `\` in docstrings: https://juliadocs.github.io/Documenter.jl/stable/man/latex/#Escaping-Characters-in-Docstrings

````suggestion
The viscous dissipation ``\\epsilon`` is defined

```math
ϵ ≡ ∫ uⱼ ∂ⱼ τᵢⱼ \\, \\rm{d} V
```
````